pkg/devfile/parser/data/common/types.go

```diff
@@ -151,6 +151,13 @@ type Exec struct {
 
 	// Working directory where the command should be executed
 	WorkingDir string `json:"workingDir,omitempty" yaml:"workingDir,omitempty"`
+
+	// +optional
+	// Whether the command is capable to reload itself when source code changes.
+	// If set to `true` the command won't be restarted and it is expected to handle file changes on its own.
+	//
+	// Default value is `false`
+	HotReloadCapable bool `json:"hotReloadCapable,omitempty" yaml:"hotReloadCapable,omitempty"`
```

Comment:
IMO we need doc updation also for HotReloadCapable behaviour.

Reply:
Already mentioned in PR desc that i would raise a separate PR for doc

Reply:
I didn't see that anywhere in the PR description 👀 

I would say adding the doc with this PR itself makes more sense than in other pr. I really think that to be a part of this issue. TBH its a better practice to get everything included in one PR. So, can you please add it in this PR itself.

Reply:
Added.